theia-php-docker/next.package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia PHP Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/callhierarchy": "next",
+        "@theia/editor-preview": "next",
+        "@theia/file-search": "next",
+        "@theia/getting-started": "next",
+        "@theia/git": "next",
+        "@theia/json": "next",
+        "@theia/markers": "next",
+        "@theia/merge-conflicts": "next",
+        "@theia/messages": "next",
+        "@theia/mini-browser": "next",
+        "@theia/navigator": "next",
+        "@theia/outline-view": "next",
+        "@theia/plugin-ext-vscode": "next",
+        "@theia/preferences": "next",
+        "@theia/preview": "next",
+        "@theia/search-in-workspace": "next",
+        "@theia/terminal": "next",
+        "@theia/textmate-grammars": "next"
+    },
+    "devDependencies": {
+        "@theia/cli": "next",
+        "@theia/debug": "next"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
+        "php-intellisense": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/felixfbecker/extension/php-intellisense/2.3.5/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage", 
```

Comment:
It'd be nice to reference the `releases` instead of the marketplace for the `php-intellisense` extension:
https://github.com/felixfbecker/vscode-php-intellisense/releases

Reply:
If it was not clear, I mean point to the `.vsix` file from the releases page, for example:
https://github.com/felixfbecker/vscode-php-intellisense/releases/download/v2.3.13/php-intellisense.vsix